faf.spec.in

```diff
@@ -56,7 +65,11 @@ BuildRequires: python-psycopg2 >= 2.5
 BuildRequires: python-testing.postgresql < 1.2
 BuildRequires: rpm-python
 BuildRequires: python-sqlalchemy >= 0.8
+%if 0%{?with_dnf}
```

Comment:
The condition should look like %if %{with dnf} - see the documentation http://rpm.org/user_doc/conditional_builds.html

Reply:
the same in other cases.

Reply:
Thx, it will be updated be fixed in my next commit.